### PR TITLE
feat: Dockerfile for ArgoCI

### PR DESCRIPTION
**Pull request by greut**
A simple dockerfile that will be used to create the chorus-backend image in our registry.

```
$ docker buildx build --tag chorus-backend .

$ docker run --rm -v $(pwd)/configs:/configs chorus-backend
Using config file: /configs/dev/chorus.yml
...
```

The configs will be provided by the external world, and shouldn't exist here if we are to comply with the 12factors' way. https://12factor.net/config

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#syntax=docker/dockerfile:1
```
Comment (sami-sweng) [inline Dockerfile:1]: if possible move to ./docker/dockerfiles

**Review by sami-sweng — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (greut) [reply Dockerfile:1]: or, how should it be called?

**Comment by iDmple** [comment]
Can we merge this in priority @sami-sweng @greut? JLR wants the demo. Ty

File: Dockerfile (hunk 1 above)
Comment (sami-sweng) [reply Dockerfile:1]: Is it meant as a replacement for stage2? If so stage2.

Comment (greut) [reply Dockerfile:1]: @sami-sweng it's meant to replace the whole `docker` folder.

Only replacing stage2 seems to make stage1 moot. Does it sound correct?

**Comment by greut** [comment]
> Can we merge this in priority @sami-sweng @greut? JLR wants the demo. Ty

There is more to the demo stage than that. It's one of the prerequisites.

File: Dockerfile (hunk 1 above)
Comment (sami-sweng) [reply Dockerfile:1]: The stage 1 will still be used in the CI and there is the dev one.

Let's keep it, there is never only one dockerfile.

Comment (greut) [reply Dockerfile]: > Let's keep it, there is never only one dockerfile.

let's call it. `docker/dockerfiles/Dockerfile` then.

**Comment by greut** [comment]
each push to master will be built with a version made up; we can change to proper tags once you're ready.

**Pull request merged**